Sources/LeafKit/LeafLexer/LeafToken.swift

```diff
@@ -202,6 +202,42 @@ public indirect enum ParameterDeclaration: CustomStringConvertible {
             case .tag:        return "tag"
         }
     }
+    
+    func imports() -> Set<String> {
+        switch self {
+            case .parameter(_): return .init()
+            case .expression(let e): return e.imports()
+            case .tag(let t):
+                guard t.name == "import" else { return t.imports() }
+                guard let parameter = t.params.first,
+                      case .parameter(let p) = parameter,
+                      case .stringLiteral(let key) = p,
+                      key.count > 0 else { return .init() }
+                return .init(arrayLiteral: key)
+        }
+    }
+    
+    func inlineImports(_ imports: [String : Syntax.Export]) -> ParameterDeclaration {
+        switch self {
+            case .parameter(_): return self
+            case .tag(let t):
+                guard t.name == "import" else {
+                    return .tag(.init(name: t.name, params: t.params.inlineImports(imports)))
+                }
+                guard let parameter = t.params.first,
+                      case .parameter(let p) = parameter,
+                      case .stringLiteral(let key) = p,
+                      key.count > 0,
```

Comment:
NitPick: `!key.isEmpty`

Reply:
Removed since the `let export = imports[key]?...` would catch an empty set anyway